docs/sdk/tutorials/export_a_kili_project.md

```diff
@@ -1,22 +1,26 @@
 # How to export a Kili project
 ## Outline
-This tutorial explains the multiple ways to export a Kili project. It describes a per-label method involving label filtering and conversion, and also the solutions performing a full-project export. The methods are illustrated with code snippets.
+This tutorial explains the multiple ways to export a Kili project. It describes:
+ * A per-label method involving label filtering and conversion
+ * The solutions for performing a full-project export.
+The methods are illustrated with code snippets.
+
```

Comment:
![image](https://user-images.githubusercontent.com/15847892/211299058-c9b06f0f-fb19-44d3-954a-8c29b36f0b0b.png)

make sure you render the doc `mkdocs serve` and check the render before submitting a PR

Reply:
fixed